package.json

```diff
@@ -29,7 +29,8 @@
     "build-prepack": "babel src --out-dir lib --source-maps",
     "watch": "babel src scripts --out-dir lib --watch --source-maps",
     "lint": "eslint src scripts",
-    "flow": "flow version; flow check --merge-timeout 0",
+    "flow": "flow version; flow status",
```

Comment:
Instead could we do:

```
"flow": "flow",
```

Since `flow status` is the default behavior. This enables you to run whatever args you want:

```
yarn flow check
yarn flow cycle
yarn flow version
```